server/graphql/v2/mutation/ConnectedAccountMutations.ts

```diff
@@ -37,20 +38,29 @@ const connectedAccountMutations = {
         throw new Unauthorized("You don't have permission to edit this collective");
       }
 
-      if (args.connectedAccount.service === Service.TRANSFERWISE) {
+      if ([Service.TRANSFERWISE, Service.PAYPAL].includes(args.connectedAccount.service)) {
         if (!args.connectedAccount.token) {
-          throw new ValidationFailed('A token is required for TransferWise accounts');
+          throw new ValidationFailed('A token is required');
         }
         const sameTokenCount = await models.ConnectedAccount.count({
-          where: { service: Service.TRANSFERWISE, token: args.connectedAccount.token },
+          where: { token: args.connectedAccount.token },
```

Comment:
Not for this PR, but as a follow up we might want to add some rate limit on this endpoint to prevent bruteforce attacks on tokens, ie:

```
> createConnectedAccount(token: "aaaaaa")
Ok
> createConnectedAccount(token: "aaaaab")
Ok
> createConnectedAccount(token: "aaaaac")
This token is already being used ==> Found a token
```